src/UI/class-recommended-widget.php

```diff
@@ -88,14 +117,15 @@ private function get_api_url( string $api_key, ?int $published_within, ?string $
 	/**
 	 * This is the widget function.
 	 *
-	 * @param array $args Widget Arguments.
-	 * @param array $instance Values saved to the db.
+	 * @param array<string, string> $args            Widget Arguments.
+	 * @param array<mixed>          $widget_settings Values saved to the db.
 	 */
-	public function widget( $args, $instance ): void {
+	public function widget( $args, $widget_settings ): void /* @phpstan-ignore-line */ {
```

Comment:
- Changed var name because we have to use `get_widget_settings` function below which will either use our values or default. So used `instance` there so that we will have less changes in the code (if we use a separate name then every occurrance of `instance` will need to be changed.
- Uses `/* @phpstan-ignore-line */` here because our function signature have better types than `WP_Widget::widget` functions and compiler were complaining so going with more type safe option